internal/mempool/mempool.go

```diff
@@ -874,7 +875,7 @@ func (txmp *TxMempool) purgeExpiredTxs(blockHeight int64) {
 
 func (txmp *TxMempool) notifyTxsAvailable() {
 	if txmp.Size() == 0 {
-		panic("attempt to notify txs available but mempool is empty!")
```

Comment:
was this an issue we saw?

Reply:
It was not an issue, but this change is to prevent a future possible issue to align with the latest tendermint repo: https://github.com/tendermint/tendermint/blob/main/mempool/v1/mempool.go#L757

I don't think it is necessary to panic just for txmp is empty